app/src/main/ViewsManifest.xml

```diff
@@ -0,0 +1,5 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Suggest also adding a TODO in these temporary manifests to remove them once the libraries can be split up.

Reply:
Okay!